compliance/lucene/src/test/java/org/eclipse/rdf4j/sail/lucene/LuceneSailTest.java

```diff
@@ -17,7 +17,7 @@
 
 	/*
 	 * (non-Javadoc)
-	 * 
```

Comment:
I'm sorry about all of these. Intellij feels the there shouldn't be a trailing space, and our style guide has no rule for this. I think Eclipse uses trailing spaces for javadocs.

Reply:
Fwiw I agree with IntelliJ here. I'll take a look at my eclipse settings to see if I can get it to remove these trailing whitespaces.